openid/configurationprovider_test.go

```diff
@@ -37,14 +39,15 @@ func Test_getConfiguration_UsesCorrectUrl(t *testing.T) {
 func Test_getConfiguration_WhenGetReturnsError(t *testing.T) {
 	c := NewHTTPClientMock(t)
 	configurationProvider := httpConfigurationProvider{getConfig: c.httpGet}
+	req := httptest.NewRequest(http.MethodGet, "/", nil)
```

Comment:
the test above will already validate that you are passing the correct request in getConfiguration so for all the other tests you can continue to pass 'nil' like before and continue the validation.